src/apps/backend/modules/account/rest_api/account_view.py

```diff
@@ -27,8 +29,35 @@ def get(self, id) -> ResponseReturnValue:
       account = AccountService.get_account_by_id(params=account_params)
       account_dict = asdict(account)
       return jsonify(account_dict), 200
+
+    except AccountNotFoundError as exc:
+      return jsonify({
+        "message": exc.message,
+        "code": exc.code,
+      }), 400
+      
+  def patch(self, id) -> ResponseReturnValue:
+    try:
+      request_data = request.get_json()
+      reset_account_params = ResetPasswordParams(account_id=id, **request_data)
+      account = AccountService.reset_account_password(params=reset_account_params)
+      account_dict = asdict(account)
+      return jsonify(account_dict), 200
+    
     except AccountNotFoundError as exc:
+      return jsonify({
+        "message": exc.message,
+        "code": exc.code,
+      }), 404
```

Comment:
This approach does not looks right to me at all. According to this - We would have to catch and serialize every error, right?

We should find a way to have a common error handler which should be able to handle a general implementation of the error (example - We got `AppError` here, we know where the http status code and message would be).

Take a look at how it's being done in MERN currently [here](https://github.com/jalantechnologies/boilerplate-mern/blob/main/src/apps/backend/modules/application/application-server.ts#L37).

Reply:
Fixed!!